ksqldb-serde/src/main/java/io/confluent/ksql/serde/json/KsqlJsonDeserializer.java

```diff
@@ -126,7 +126,7 @@ public T deserialize(final String topic, final byte[] bytes) {
       return SerdeUtils.castToTargetType(coerced, targetType);
     } catch (final Exception e) {
       throw new SerializationException(
-          "Failed to deserialize " + target + " from topic: " + topic, e);
+          "Failed to deserialize " + target + " from topic: " + topic + ". " + e.getMessage(), e);
```

Comment:
What's the philosophy behind when it makes sense to copy the underlying error message into the new error message?

Reply:
Humm.. hard to give a hard and fast rule, but I generally make this change if otherwise the user would be presented with a less than useful error message.  In this case, I added it to make debugging easier... both for us and for users interacting with the processing logger.  Having the cause's message as part of the top level message means you can quickly understand what's happening without having to scroll through exception call stacks.